<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>380</y>
      <width>471</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QFrame" name="frame_4">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButtonZurueck">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Zurück</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonAdd">
          <property name="text">
           <string>Hinzufügen</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonWeiter">
          <property name="text">
           <string>weiter</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelAnzahl">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Eintrag text</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QWidget" name="layoutWidget_3">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>90</y>
          <width>171</width>
          <height>111</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="labelVorname">
           <property name="text">
            <string>Vorname</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditVorname"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="labelNachname">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>90</y>
          <width>169</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string>Nachname</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditNachname">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>177</y>
          <width>169</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_3">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="labelTelefonnummer">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>93</y>
          <width>169</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string>Telefonnummer</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditTelefonnummer">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>180</y>
          <width>169</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Datei">
    <property name="title">
     <string>&amp;Datei</string>
    </property>
    <addaction name="action_Neu"/>
    <addaction name="action_Speichern"/>
    <addaction name="action_Laden"/>
    <addaction name="action_Beenden"/>
   </widget>
   <addaction name="menu_Datei"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Neu">
   <property name="text">
    <string>&amp;Neu</string>
   </property>
  </action>
  <action name="action_Speichern">
   <property name="text">
    <string>&amp;Speichern</string>
   </property>
  </action>
  <action name="action_Laden">
   <property name="text">
    <string>&amp;Laden</string>
   </property>
  </action>
  <action name="action_Beenden">
   <property name="text">
    <string>&amp;Beenden</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
